plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryTypeMapping.java

```diff
@@ -526,44 +533,122 @@ public void testTime()
                 .execute(getQueryRunner(), bigqueryViewCreateAndInsert("test.time"));
     }
 
-    @Test
-    public void testTimestampWithTimeZone()
+    @Test(dataProvider = "zonesDataProvider")
```

Comment:
The data provider name is wrong. Could you run this test before pushing a change?

Reply:
How do you run these tests locally?

Reply:
runnning `./mvnw clean install -B --strict-checksums -V -T 1C -pl ":trino-bigquery"` but don't seem to be included.

Reply:
There's another profile `cloud-tests`.

Reply:
totally missed that. Thanks

Reply:
ok I believe I understand things a bit better. Sorry I've had limited time to spend on this. Thanks for the reviews and patience.